### ISSUE TITLE
Document/Improve Account Recovery Workflow

### ISSUE THREAD

**Issue by dmitrizagidulin** [solid/solid#19]
1. Document the account recovery (and new certificate generation) API endpoints for Gold and LDNode (See also https://github.com/linkeddata/gold/issues/68, for example).

2. On the 'enter your WebID' recovery screen, prompt the user for what their WebID looks like (e.g. `https://accountname.databox.me/profile/card#me`)

3. On the 'enter your WebID' recovery screen, display an error message when the WebID is not valid (does not point to a valid profile document).

4. Display an error message when there is no recovery email on file.

5. Improve the 'issue a new certificate' landing page (that the user goes to when clicking the link in the email).

I suspect this is currently a "hardcoded" app on each Solid server. Perhaps we should consider this to be a separate recovery app with its own Github repo.

**Comment by jwhite66** [comment]
Just thought I'd drop an outsiders perspective:  the lack of any clarity around WebID management on databox is pretty much defeating my attempts to engage with and learn Solid.  Not sure how to use the GitHub issue system to suggest raising priority, so I'll just do it noisily in the comments :-/.

**Comment by melvincarvalho** [comment]
@jwhite66 thanks for dropping in a comment, it's great to hear the perspective of someone learning the system!

A webid is simply another name for a "userURI"  ie an http URI that belongs to that user.

Solid is a work in progress, and we would love to improve, but bear in mind we have limited resources, some of whom are volunteers.  

Is your concert around account recovery, webid managment, general lack of documentation or a user experience that is hard to follow.  What do you think could be improved?

**Comment by jwhite66** [comment]
Sorry, my comment was unclear.  I was trying to say that the information already contained in this issue would be useful if exposed live.  So I was trying to suggest this issue get a boost in priority.  But I think maybe I'll go be naive on chat instead of here...
